src/jvmMain/kotlin/generated/model/db/DvdRentalDb.kt

```diff
@@ -295,31 +295,31 @@ object DvdRentalDb {
         source[Table.specialFeatures], source[Table.fullText])
     fun fetchAll() = transaction { with (Table) { selectAll().map { create(it) } } }
     object Table : IntIdTable("Film") {
-      val filmId: Column<Int> = integer("film_id")
+      val filmId: Column<Int> = integer("filmId")
 
       val title: Column<String> = text("title")
 
       val description: Column<String> = text("description")
 
-      val releaseYear: Column<Int> = integer("release_year")
+      val releaseYear: Column<Int> = integer("releaseYear")
 
-      val languageId: Column<Int> = integer("language_id")
+      val languageId: Column<Int> = integer("languageId")
 
-      val rentalDuration: Column<Int> = integer("rental_duration")
+      val rentalDuration: Column<Int> = integer("rentalDuration")
 
-      val rentalRate: Column<Double> = double("rental_rate")
+      val rentalRate: Column<Double> = double("rentalRate")
 
       val length: Column<Int> = integer("length")
 
-      val replacementCost: Column<Double> = double("replacement_cost")
+      val replacementCost: Column<Double> = double("replacementCost")
 
       val rating: Column<String> = text("rating")
 
-      val lastUpdate: Column<String> = text("last_update")
+      val lastUpdate: Column<String> = text("lastUpdate")
 
-      val specialFeatures: Column<String> = text("special_features")
```

Comment:
You need to add back the db namespace so you can have a name and db-specific type.